src/ui/configuration/Radio3DRSettings.cc

```diff
@@ -738,7 +734,7 @@ void Radio3DRSettings::writeEscapeSeqeunceNow()
 void Radio3DRSettings::readLocalVersionString()
 {
     QLOG_DEBUG() << "Radio3DRSettings::readLocalVersionString()";
-    emit updateLocalStatus(tr("Read local radio version"), black);
+    emit updateLocalStatus(tr("Read local radio versionDDDDDDDDDDD"), black);
```

Comment:
Are those DDDD intended?

Reply:
Must be a new coding standard or something... I'll try to fix it ASAP

Reply:
It should be ok now.